gatsby-config.js

```diff
@@ -93,5 +94,7 @@ module.exports = {
         pathToConfigModule: "src/utils/typography",
       },
     },
+    "gatsby-plugin-now",
```

Comment:
Where it's being used?

Reply:
It makes redirects work. Since I removed the redirects I can remove this too.